cetmix_tower_server_monitor/models/cx_tower_server.py

```
@@ -0,0 +1,579 @@
+
+    # Format JSON manually
+    PAYLOAD='{{"server_ref": "'$REF'", "token": "'$TOKEN'", "metrics": {{'
+    PAYLOAD+='"ram_total_mb": '$MEM_TOTAL', "ram_used_mb": '$MEM_USED', '
+    PAYLOAD+='"disk_total_gb": '$(awk "BEGIN {{print $DISK_TOTAL/1024}}")', "disk_used_gb": '$(awk "BEGIN {{print $DISK_USED/1024}}")', '
+    PAYLOAD+='"cpu_percent": '$CPU_USAGE', "cpu_cores": '$CPU_CORES' }}}}'
+
+    # Send to Tower and get next interval
+    RESPONSE=$(curl -s -X POST -H "Content-Type: application/json" -d "$PAYLOAD" "$URL")
+
+    # Simple JSON extraction for next_interval
+    NEXT=$(echo "$RESPONSE" | grep -oP '"next_interval":\s*\K\d+')
```

Pylint (4.0.5)

[warning] Anomalous backslash in string: '\s'. String constant might be missing an r prefix.

(W1401)